app/assets/stylesheets/styles/components/guides/_new_guides.css.scss

```diff
@@ -22,6 +22,148 @@
 }
 
 .new-guide {
+<<<<<<< HEAD
+
+  .guide-creation-nav {
+    $guide-nav-color: #5E9546;
+    background: $guide-nav-color;
+    overflow-x: auto;
+    overflow-y: hidden;
+
+    /* the ul containing the links */
+    ul {
+      list-style: none;
+      margin-left: 0;
+      padding-top: 5px;
+      padding-left: 0;
+      padding-right: 0;
+      text-align: center;
+      white-space: nowrap;
+    }
+
+    li {
+      position: relative;
+      display: inline-block;
+    }
+
+    li a {
+      color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.